java/build/bazel/tests/integration/WorkspaceDriver.java

```diff
@@ -66,7 +72,9 @@ private static void setupRepositoryCache() throws IOException {
         repositoryCache.put(Paths.get(dep));
       }
     }
-    // repositoryCache.freeze();
+    //freeze doesn't work on Windows
+    if (OS.getCurrent() != OS.WINDOWS)
+      repositoryCache.freeze();
```

Comment:
Should we indicate to the caller that freezing is not supported in some way?

Reply:
Javadoc? Readme? Both? I don’t like these methods but I’m not sure where else since this happens pretty implicitly to the user so it’s not trivial to propagate it back up in the method name